src/Keboola/Google/BigQuery/RestApi/Job.php

```diff
@@ -120,6 +120,26 @@ private function validateStatus(Response $response, Client $client)
         return true;
     }
 
+    private function getJobUrl(array $responseBody)
+    {
+        if (empty($responseBody['jobReference']['projectId'] || $responseBody['jobReference']['jobId'])) {
```

Comment:
Nerozpisal by si to radsej na `&&`? ... akoze `empty(x) && empty(y)`

Reply:
jj ma to byt `&&`

Reply:
Ok, tak to je jedine, co mi prislo divne.

Reply:
Ale taram, sorry. Ma to byt `empty(x) || empty(y)`

Reply:
Len sa mi nezda, ze je to v kope :)

Reply:
jj vubec nechapu jak jsme tu podminku vytvoril